pymysqlreplication/tests/benchmark.py

```diff
@@ -25,12 +25,12 @@ def consume_events():
         only_events=[UpdateRowsEvent],
         only_tables=["test"],
     )
-    start = time.clock()
+    start = time.perf_counter()
```

Comment:
perf_counter method is it possible 3.3, 3.4, 3.5 and 3.6?

Reply:
> python

https://docs.python.org/3/library/time.html#time.perf_counter

yes it says this method was introduced in python 3.3